examples/chat/Cargo.toml

```diff
@@ -6,8 +6,17 @@ publish = false
 license = "MIT"
 
 [dependencies]
-async-std = { version = "1.12", features = ["attributes"] }
+tokio = { version = "1.32", features = ["full"] }
 async-trait = "0.1"
 env_logger = "0.10.0"
 futures = "0.3.28"
-libp2p = { path = "../../libp2p", features = ["async-std", "gossipsub", "mdns", "noise", "macros", "tcp", "yamux", "quic"] }
+libp2p = { path = "../../libp2p", features = [
```

Comment:
Can you undo this formatting please?

Reply:
okay, was some automatic code formatting